CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Flyte Console Contribution Guide
```

Comment:
I believe this file needs to be further simplified for first-time contributors. If possible, we need to have a step-by-step guide explaining the contribution process. Also, please check if the current content fits into the contribution guide. :)